client/src/pages/index.tsx

```diff
@@ -276,19 +269,25 @@ const Homepage: NextPage<FeedsPageInitials> = (props): JSX.Element => {
 						}}
 					>
 						<FlowHeader
-							openFilterModal={(): void => setDisplayFilterModal(true)}
-							setSortBy={(val: 'top' | 'hot' | undefined): void => handleSortBySet(val)}
-							resetCategoryFilter={(): void => handleCategoryFilterSet(undefined)}
+							setSortBy={(val: 'top' | 'hot' | undefined): void => handleSorting(val)}
+							resetTagFilter={(): void => setTagFilter(null)}
+							beforeFilterReset={() => handleFlowReset()}
```

Comment:
The anonymous function is useless.
```suggestion
							beforeFilterReset={handleFlowReset}
```